libgraph/src/GraphTopology.cpp

```diff
@@ -526,7 +526,7 @@ katana::CondensedTypeIDMap::MakeFromEdgeTypes(
   katana::do_all(
       katana::iterate(Edge{0}, topo.NumEdges()),
       [&](const Edge& e) {
-        katana::EntityTypeID type = pg->GetTypeOfEdgeFromPropertyIndex(e);
+        katana::EntityTypeID type = pg->GetTypeOfEdgeFromTopoIndex(e);
```

Comment:
That's because this loop simply iterates over the edges in a topology from 0 to max value, which is the topology index. Essentially, it means the index into an adjacency vector.

Reply:
OK, got it. I hope we can put this ugliness behind us soon.